jina/parsers/orchestrate/runtimes/remote.py

```diff
@@ -145,7 +145,9 @@ def _add_host(arg_group):
         '--host',
         type=str,
         default=__default_host__,
-        help=f'The host address of the runtime, by default it is {__default_host__}.',
+        help=f'The host address of the runtime, by default it is {__default_host__}.'
+        ' In the case of an external Executor (`--external` or `external=True`) this can be a list of hosts, separated by commas.'
+        ' Then, every resulting address will be considered as one replica of the Executor.',
```

Comment:
I find it weird that other components like the client share the exact same arg here. The extra description doesn't really apply to the client. I think we can either follow pattern [here](https://github.com/jina-ai/jina/blob/master/jina/parsers/__init__.py#L91) or make clone this parameter in client args with different description.
PS: we can handle this in separate PR especially if we have many more similar args

Reply:
I think this is an issue in many places. I am in favour of tackling this separately